.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ]  [OPTIONAL] since this is a personal vs code settings file you can add it to your .gitignore file.